### ISSUE TITLE
Routings anpassen

### ISSUE THREAD

**Issue by BugBuster1701** [BugBuster1701/contao-visitors-bundle#21]
aus der Doku: https://symfony.com/doc/current/bundles/best_practices.html

> Routing
> 
> If the bundle provides routes, they must be prefixed with the bundle alias. For example, if your bundle is called AcmeBlogBundle, all its routes must be prefixed with acme_blog_.

**Comment by Coding-Kiwi** [comment]
Version 1.3.5 schmeißt bei mir den Error
"Unable to generate a URL for the named route "visitors_frontend_screencount" as such route does not exist." hat das mit diesem Issue hier zu tun?

**Comment by BugBuster1701** [comment]
Nein, hier gehts um eine Umbenennung um mehr in Richtung "Best Practise" zu kommen.

**Comment by BugBuster1701** [comment]
Weiter zu deinem Problem im Ticket #50
